### DISCUSSION TITLE
Toggling Codelens cleanly

### DISCUSSION THREAD

**Discussion by shmup** [neoclide/coc.nvim#4982] [Q&A] [answered]
My problem with codelens is the noise it adds to a buffer. But it's also very useful, when I need it. I find myself leaving it off by default, and toggling the setting to `true` on demand, which is great and doing so will automatically update a buffer with its reference/implementation count.

It's turning codelens back off I want to solve (aka, hiding the counts). The only solution I've found is to delete the buffer with `:bd`, where `CocRestart` doesn't work (nor does it seem like the right idea)

I may be overlooking something.

Cheers

**Comment by fannheyward** [comment, marked as answer]
Try `:CocCommand document.toggleCodeLens`.
